### DISCUSSION TITLE
Motion controlls?

### DISCUSSION THREAD

**Discussion by SamsungHandy** [nefarius/DsHidMini#173] [Support]
so im playing yuzu which works great with ds4, ds3 etc. but my  ds3 i cant seem to get working with motion controls is it possible with bthps3 and dshidmini?

**Comment by nefarius** [comment]
Nope, not implemented.

**Reply by SamsungHandy** [reply]
is there a different way to get motion controls? or why its not implemented?

**Reply by nefarius** [reply]
I explained it a million times in the past but here we go again: I don't personally care about sixaxis motion controls and don't know how to transform and calibrate them, therefore: not implemented 😉 So unless someone else does it, it will never be available. Simple.

**Comment by Ryochan7** [comment]
No point in doing so because of the hardware. The DS3 gyro only handles one axis (Yaw). Only really useful for shake commands. Can't provide any means of decent camera control without a second axis (Pitch) to work with.

Going to have to use a different controller with an updated IMU (DS4, Steam Controller, Switch Pro, Joy-Con, etc).

**Comment by GuiCAlves** [comment]
You can use the older tool, the SCPToolkit with a patch made by the person who developed Cemuhook (https://github.com/rajkosto), here is the link with a tutorial, https://cemuhook.sshnuke.net/padudpserver.html.

I think dshidmini is a much better tools, but to play Mario Odyssey with a DS3, SCPToolkiot is the way to go.

Maybe @nefarius can take a look on the patch made by rajkosto to see if it can be ported to dshidmini.

**Reply by nefarius** [reply]
No. You can't just port over mixed partially closed source C/C# code over to a driver, not possible. Please let the topic rest, I will not invest any more time into it.
